velox/functions/prestosql/GreatestLeast.cpp

```diff
@@ -112,6 +112,15 @@ class ExtremeValueFunction : public exec::VectorFunction {
       exec::EvalCtx& context,
       VectorPtr& result) const override {
     switch (outputType.get()->kind()) {
+      case TypeKind::INTEGER:
+        if (isDateType(outputType)) {
```

Comment:
This is confusing.... I think if TypeKind::INTEGER should just call applyTyped... The check for Date type is not needed here.

Reply:
Got it, since the original check was for Date type only, this check was added. Reverted now, thanks!